Classes/Command/CacheCommandController.php

```diff
@@ -122,4 +129,25 @@ public function listGroupsCommand()
                 break;
         }
     }
+
+    /**
+     * Flush cache for $cacheType with data handler implementation
+     *
+     * @param string $cacheType either "system" or "all"
+     * @return void
+     * @throws \UnexpectedValueException
+     */
+    public function flushCacheWithDataHandlerHandlerCommand($cacheType)
```

Comment:
Implementing the functionality in the cache service is fine with me. I will go for that.

- actually both hooks, `clearCachePostProc` is more important to us, as we use it in projects
- from a user perspective it might be a bit hard to explain, but we actually do have a couple of examples inside TYPO3 where you can do almost the same thing using different methods.
- I'm not sure if we can `just call the hooks` cause we can't be assured how other have implemented them
- `clearPageCacheEval` hook, I have to dig into that